package/yast2-storage-ng.spec

```diff
@@ -30,6 +30,8 @@ Requires:	yast2-ruby-bindings >= 4.0.6
 Requires:	libstorage-ng-ruby >= 3.3.254
 # communicate with udisks
 Requires:	rubygem(ruby-dbus)
+# Y2Packager::Repository
+Requires:	yast2-packager >= 3.3.7
```

Comment:
Hmmm, yast2-packager depends on yast2-storage-ng. Is it safe/good to have such a cycle? Just asking.

Reply:
in Requires it is not problem. You have to avoid it in BuildRequires

Reply:
In fact, `Y2Packager::Repository` can be mocked during tests, so I will remove BuildRequires one.